cli/state.rs

```diff
@@ -230,6 +230,23 @@ impl State {
       dispatcher(isolate, &state, args, zero_copy)
     }
   }
+
+  /// Quits the process if the --unstable flag was not provided.
+  ///
+  /// This is intentionally a non-recoverable check so that people cannot probe
+  /// for unstable APIs from stable programs.
+  pub fn check_unstable(&self, api_name: &str) {
+    // TODO(ry) Maybe use IsolateHandle::terminate_execution here to provide a
```

Comment:
Stack trace from `TerminateExecution` has no meaningful info - better to return error to JS (by adding new error kind) and throw it there

Reply:
I don't want it to be recoverable

Reply:
I agree - otherwise we will get modules that progressively enhance for `unstable` things. I think that is an antipattern.